resilience4j-retry/src/test/java/io/github/resilience4j/retry/RetryConfigBuilderTest.java

```diff
@@ -168,6 +181,13 @@ public void shouldBuilderCreateConfigEveryTime() {
         assertThat(config1.getMaxAttempts()).isEqualTo(5);
     }
 
+    @Test(expected = IllegalStateException.class)
+    public void intervalFunctionUsedWithIntervalBiFunctionShouldFail() {
```

Comment:
Could you please add a final test that if neither intervalBiFunction nor intervalFunction is set, that the default setting is used in the RetryConfig?